ui/app/routes/vault/cluster/dashboard.js

```diff
@@ -22,14 +22,18 @@ export default class VaultClusterDashboardRoute extends Route {
   }
 
   model() {
-    const versionHeader = this.version.isEnterprise
-      ? `Vault v${this.version.version.slice(0, this.version.version.indexOf('+'))}`
-      : `Vault v${this.version.version}`;
+    const vaultConfiguration = this.getVaultConfiguration();
 
     return hash({
-      versionHeader,
+      vaultConfiguration,
       secretsEngines: this.store.query('secret-engine', {}),
-      vaultConfiguration: this.getVaultConfiguration(),
     });
   }
+
+  setupController(controller, resolvedModel) {
```

Comment:
I'm not sure I understand adding `vaultConfiguration` to the controller, checking for just a `number` feels a bit odd, is the number an error code? If we're not concerned with handling errors for the config, then it seems like we should just return `false` or `null` in `getVaultConfiguration()` and then in the template just check for `{{#if @model.vaultConfiguration}}`

But I'm not super confident in my controller knowledge!

Reply:
I love that idea!